src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -230,17 +230,10 @@
  *   &lt;property name=&quot;option&quot; value=&quot;bottom&quot;/&gt;
  *   &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
- * </pre>
- * <pre>
- * &lt;?xml version=&quot;1.0&quot;?&gt;
```

Comment:
A few lines up has the following text:
>  Note: a <a href="config_filters.html#SuppressionFilter">suppression filter</a> is needed because
>  IDEA has no blank line between "javax" and "java".

Since it is converted to SuppressionXpathSingleFilter the text needs to be updated.

Reply:
@rnveach So change the text and the link of suppression filter into the suppressionxpathsinglefilter?

Reply:
Yes

Reply:
Done.